doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -0,0 +1,361 @@
+========================
+ mClock Config Reference
+========================
+
+.. index:: mclock; configuration
+
+This dmclock related configuration parameters can be applied using mclock
+profiles. The purpose of the mclock profiles is to mask the low level details
+from a user trying to configure mclock for a specific purpose. At a high level,
+for each OSD in the enviroment the steps involve,
+
+- providing an input related to the total capacity of OSD(s) and
```

Comment:
we could add a reference to this section https://ceph--40531.org.readthedocs.build/en/40531/rados/configuration/mclock-config-ref/#determine-osd-capacity-using-benchmark-tests, to help the reader understand what capacity means here